src/components/slider/slider.js

```diff
@@ -382,29 +382,63 @@ function SliderDirective($$rAF, $window, $mdAria, $mdUtil, $mdConstant, $mdThemi
 
     /**
      * left/right/up/down arrow listener
+     * @param {!KeyboardEvent} ev
      */
     function keydownListener(ev) {
       if (isDisabled()) return;
+      var keyCodes = $mdConstant.KEY_CODE;
 
       var changeAmount;
-      if (vertical ? ev.keyCode === $mdConstant.KEY_CODE.DOWN_ARROW : ev.keyCode === $mdConstant.KEY_CODE.LEFT_ARROW) {
-        changeAmount = -step;
-      } else if (vertical ? ev.keyCode === $mdConstant.KEY_CODE.UP_ARROW : ev.keyCode === $mdConstant.KEY_CODE.RIGHT_ARROW) {
-        changeAmount = step;
+      switch (ev.keyCode) {
+        case keyCodes.DOWN_ARROW:
+        case keyCodes.LEFT_ARROW:
+          ev.preventDefault();
+          changeAmount = -step;
+          break;
+        case keyCodes.UP_ARROW:
+        case keyCodes.RIGHT_ARROW:
+          ev.preventDefault();
+          changeAmount = step;
+          break;
+        case keyCodes.PAGE_DOWN:
+          ev.preventDefault();
+          changeAmount = -step * 4;
```

Comment:
Why is the larger step, `step * 4`?  4 seems to be an arbitrary value.

Reply:
In the ARIA practices examples they jump 10 steps:
https://www.w3.org/TR/wai-aria-practices-1.1/examples/slider/slider-1.html
https://www.w3.org/TR/wai-aria-practices-1.1/examples/slider/slider-2.html

But the spec only says:
> Page Up (Optional): Increment the slider by an amount larger than the step change made by Up Arrow.

We could make this configurable, but I would prefer to do that in a separate issue and PR.

Reply:
We also support moving 4 steps by pressing the meta/alt/ctrl key along with the arrow key. For now this step size is the same as paging.

Do you think that they should be different?
- arrow + modifying = 4 steps
- page up/down = 10 steps?

Reply:
The arrow + modifier is pre-existing behavior: https://github.com/angular/material/pull/11517/files#diff-19f59514041130aaa9c83ecc89c23536R425

Reply:
I've made the change to 4 and 10 steps as mentioned above. I've also updated the docs.